cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -42,9 +42,9 @@ if(ANDROID OR RASPBERRY_PI OR OPENWRT)
   set(configure_command AR=${CMAKE_AR} RANLIB=${CMAKE_RANLIB} "./Configure")
 elseif(MINGW)
   if(HUNTER_OPENSSL_MINGW64)
-    set(configure_command "./Configure" mingw64)
+    set(configure_command AR=${CMAKE_AR} RANLIB=${CMAKE_RANLIB} "./Configure" mingw64)
```

Comment:
`hunter_pick_archiver` should be called before because sometimes you have to use `CMAKE_CXX_COMPILER_AR` and `CMAKE_CXX_COMPILER_RANLIB` instead. See block `if(ANDROID OR RASPBERRY_PI OR OPENWRT)`.